### ISSUE TITLE
fromFormat not parsing correctly in node 12

### ISSUE THREAD

**Issue by kalvenschraut** [moment/luxon#624]
We recently upgraded to node 12 and noticed some issues in our testing environment specifically where we are trying to parse a date string with from format.

We have 
```js
DateTime.fromFormat('11/26/2019, 3:38 PM', 'f', { locale: "en" })
```
parsing correctly in node 10 and returning no error while in node 12 we are receiving an unparseable error stating our date string can not be parsed with the token f.

**Comment by kalvenschraut** [comment]
The following can be ran from a Dockerfile

```docker
FROM mhart/alpine-node:12
RUN npm i luxon
RUN echo "const { DateTime } = require('luxon'); console.log(DateTime.fromFormat('11/26/2019, 3:38 PM', 'f', { locale: 'en' }));" > test.js
RUN node test.js
```
to replicate the issue. Can change node 12 to node 10 easily by changing the first line.

**Comment by icambron** [comment]
Thanks for the report.

This appears to be because newer versions of v8 use `dayPeriod` instead of `dayperiod` in `formatToParts`. Fixed in a56a2942fc5f9b18cd5ca116bd29e92bfeedcc55, released in 1.21.3.
